devops/scripts/deploy-decoupled-upstream.sh

```diff
@@ -29,8 +29,19 @@ echo
 newcommits=$(git log decoupled-release-pointer..HEAD --reverse --pretty=format:"%h")
 commits=()
 
+# There are a small number of commits which must be manually excluded, usually
+# due to changes in the commit type rules which would retroactively include commits
+# which were not initially included, causing merge conflicts. Commits which should
+# be excluded can be added to this array and will not be cherry picked.
+exclude_list=("0099a8b")
+
 # Identify commits that should be released
 for commit in $newcommits; do
+  # Exclude commits which have been manually rejected
+  for item in "${exclude_list[@]}"; do
+    [[ $item == $commit ]] && echo "Commit has been manually excluded."; continue
```

Comment:
Adding the commit hash here would be good and in line with other messages (e.g. `ommit c2e7468 contains both release and nonrelease changes. Skipping this commit.`)

```suggestion
    [[ $item == $commit ]] && echo "Commit ${commit} has been manually excluded.";
```